renderer/views/player.js

```diff
@@ -540,3 +548,37 @@ function cssBackgroundImageDarkGradient () {
   return 'radial-gradient(circle at center, ' +
     'rgba(0,0,0,0.4) 0%, rgba(0,0,0,1) 100%)'
 }
+
+// Format the playback time of the video
+function formatPlaybackProgress (currentTime, duration) {
+  if (typeof currentTime !== 'number' || typeof duration !== 'number') {
```

Comment:
This won't catch `NaN` since `typeof NaN === 'number'`, so we should check for that case too.